mslib/plot.py

```diff
@@ -26,6 +26,9 @@
 import hashlib
```

Comment:
this plot.py is a new module, please add the docstrings from the example template.py